File: ListOfTours.Repository/Models/Tour.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ListOfTours.Repository.Models
{
    public class Tour
    {
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }

        public string ClientName { get; set; }

        public DateTime Date { get; set; }

        public ICollection<ExcursionSight> ExcursionSights { get; set; }
    }
}

File: ListOfTours/ClientApp/src/models/Tour.ts
import { ExcursionSight } from '../models/ExcursionSight';

export class Tour {
  constructor(
    public id: number,
    public name: string,
    public clientName: string,
    public date: Date,
    public excursionSights: ExcursionSight[]
  ) { }
}
